_posts/2019-06-06-NeuralODEs.markdown

```diff
@@ -0,0 +1,289 @@
+
+In this curriculum, we will go through all the background topics necessary to understand these models. At the end, you should be able to implement neural ODEs and apply them to different tasks.
+
+<br />
+
+# 1 Numerical solution of ODEs - Part 1
+  **Motivation**: ODEs are used to mathematically model a number of natural processes and phenomena. The study of their numerical 
+    simulations is one of the main topics in numerical analysis and of fundamental importance in applied sciences. A first step to understand Neural ODEs is their definition and basic numerical techniques to solve them.
+
+  **Topics**:
+
+  1. Initial values problems
+  2. One-step methods
+  3. Consistency and convergence
+
+  **Required Reading**:
+
+  1. Sections 12.1-4 from [An Introduction to Numerical Analysis](https://www.cambridge.org/core/books/an-introduction-to-numerical-analysis/FD8BCAD7FE68002E2179DFF68B8B7237#) (S\"uli & Mayers)
+  2. Sections 11.1-3 from [Numerical Mathematics](https://www.springer.com/us/book/9783540346586?token=holiday18&utm_campaign=3_fjp8312_us_dsa_springer_holiday18&gclid=Cj0KCQiAvebhBRD5ARIsAIQUmnlViB7VsUn-2tABSAhIvYaJgSEqmJXD7F4A7EgyDQtY9v_GeUsNif8aArGAEALw_wcB) (Quarteroni et al.)
+
+  
+  **Optional Reading**:
+
+  1. Runge-Kutta methods: Section 12.5 from (S\"uli & Mayers)
+  2. [Prof. Trefethen's class ODEs and Nonlinear Dynamics 4.2](http://podcasts.ox.ac.uk/odes-and-nonlinear-dynamics-42)
+
+  **Questions**:
+
+  1. Exercise 1 in Section 11.12 of (Quarteroni et al.)
+     <details><summary>Solutions</summary>
+     <p>
+     ...
+     </p>
+     </details>
+
+  2. Exercises 12.3,12.4, 12.7 in Section 12 of (S\"uli & Mayers)
+     <details><summary>Solution</summary>
+     <p>
+     ...
+     </p>
+     </details>
+
+  3. Consider the following method for solving \(y' = f(y)\):
+       
+     $$y_{n+1} = y_n + h*(theta*f(y_n) + (1-theta)*f(y_{n+1}))$$
+    
+     Assuming sufficient smoothness of \(y\) and \(f\), for what value of \(0 \leq\theta\leq 1\) is the truncation error the smallest? What does this mean about the accuracy of the method?
+     <details><summary>Solution</summary>
+     <p>
+     ...
+     </p>
+     </details>
+       
+
+  4. [Colab Notebook](https://colab.research.google.com/drive/1bNg-RzZoelB3w8AUQ6mefRQuN3AdrIqX)
+     <details><summary>Solution</summary>
+     <p>
+     ...
+     </p>
+     </details>
+
+**Notes**: Here is a [link](/assets/nodes_notes/week1.pdf) to our notes for the lesson. 
```

Comment:
These are quite good - you should emphasize them. Maybe put them at the top of each sectin, just below Motivation?

Reply:
done

Reply:
Can you emphasize the Notes more? They are legitimately qutiequite good. One approach is to describe briefly what’s in them and how it adds to the material (and not just include a link). That way, it stands out amongst the rest of the curriculum and also gives a taste of what’s in it and why it should be utilized.